### ISSUE TITLE
jsdoc comment is not expanding to multiple lines

### ISSUE THREAD

**Issue by Tyriar** [Microsoft/vscode#41010]
In a TS file, type "/**"

<img width="160" alt="screen shot 2018-01-01 at 9 45 53 am" src="https://user-images.githubusercontent.com/2193314/34469652-add177f8-eed8-11e7-8782-0e3d5ae0627d.png">

Press enter:

<img width="124" alt="screen shot 2018-01-01 at 9 46 01 am" src="https://user-images.githubusercontent.com/2193314/34469653-af275082-eed8-11e7-9e16-fa1729718bf0.png">

This is what I expect (and I'm pretty sure it used to do this):

<img width="96" alt="screen shot 2018-01-01 at 9 47 21 am" src="https://user-images.githubusercontent.com/2193314/34469656-cdee538a-eed8-11e7-8713-72d3a1bc2351.png">

**Comment by mjbvz** [comment]
Duplicate of https://github.com/Microsoft/TypeScript/issues/20089

**Comment by Tyriar** [comment]
@mjbvz is this blocked on https://github.com/Microsoft/TypeScript/issues/20619?

**Comment by mjbvz** [comment]
No, shouldn't be. We just need a new TS 2.7 insiders build to pick up the fix

Basically TS made a change to return `/** */` for jsdoc templates in invalid jsdoc locations . Previously they would only return real templates such as:

```
/**
 * @param foo 
 */
```

or `undefined` if the location for the request was not a valid jsdoc location.

We rely on `undefined` being returned as a  signal that we should insert the default comment template:

```
/**
 *
 */
```
